lib/roo/base.rb

```diff
@@ -160,9 +160,13 @@ def to_matrix(from_row = nil, from_column = nil, to_row = nil, to_column = nil,
     end)
   end
 
-  # call to_s method defined on subclasses
-  def inspect
-    to_s
+  def inspect(debug = false)
```

Comment:
Personally I would drop the debug argument - if people want the full results they can just call to_s instead.